components/clipboard/clipboard.module.css

```diff
@@ -0,0 +1,49 @@
+.clipboardButton {
+    position: absolute;
+    right: 15px;
+    top: 64px;
+    background: rgba(0, 0, 0, 0.5) url('/static/images/file_copy.svg') no-repeat center center;
```

Comment:
Instead of adding svg as background, inline it in the React code itself.
Use this svg code for copy

```html
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M16 1H4c-1.1 0-2 .9-2 2v14h2V3h12V1zm3 4H8c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h11c1.1 0 2-.9 2-2V7c0-1.1-.9-2-2-2zm0 16H8V7h11v14z"/></svg> 
```

then you won't need the file_copy.svg or .png